src/style.css

```
@@ -1 +1,84 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  margin: auto 0;
+  padding: 0 50px;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+a {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.logo__image {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.nav__item {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.nav__link {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  font-weight: 500px;
+  font-size: 12px;
+  list-style-type: none;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  text-transform: uppercase;
+}
+
+.nav__list :nth-child(8) {
+  margin-right: 0;
+}
```

here's how you can set those margins to `nav__item`
```suggestion
.nav__list:not(last-child) {
  margin-right: 20px;
}
```